configure.ac

```diff
@@ -683,6 +683,9 @@ if test x$repl = xyes; then
   AC_DEFINE_UNQUOTED(ENABLE_REPL, 1, "")
 fi
 
+PKG_CHECK_MODULES([X11], [x11], [lx11=1], [lx111=0])
```

Comment:
`lx111=0`? Prob wanted `lx11=0` instead

Reply:
Corrected. Sorry for the typo. (think this had no sideffects, since lx11 was undefined anyway)